backend/capellacollab/tools/models.py

```diff
@@ -26,6 +26,7 @@ class Tool(Base):
     id = Column(Integer, primary_key=True)
     name = Column(String)
     docker_image_template = Column(String)
+    readonly_docker_image_template = Column(String)
```

Comment:
The Pydantic model `ToolDockerImage` should also be updated to return `obj.readonly_docker_image_template` in the `from_orm` method.

Reply:
Also, the update route should be fixed.

Reply:
fixed.